nlp_architect/procedures/transformers/base.py

```diff
@@ -62,6 +62,8 @@ def inference_args(parser: argparse.ArgumentParser):
     """
     parser.add_argument("--model_path", default=None, type=str, required=True,
                         help="Path to pre-trained model")
+    parser.add_argument("--load_quantized_model", action="store_true",
+                        help="Load and perform Inference from saved quantized model, 'quant_pytorch_model.bin' file must exist in directory and model type must be 'quant_<model>'")
```

Comment:
why do you need a dedicated argument? and why in the base class?

Reply:
Since both the quantized model and the regular model are saved there is need for the user to indicate which model he desires to load.
I placed it in the base class since this flag can be used with each model that is able to use the quantized model. Do you think there is a better place for it?